src/api/spec/features/beta/webui/maintenance_workflow_spec.rb

```diff
@@ -0,0 +1,126 @@
+    end
+    update_project_branch = Project.last
+    target_packages.each do |package|
+      Backend::Connection.put("/source/#{package[:data][:targetproject]}/#{package[:data][:targetpackage]}/DUMMY_FILE", 'dummy')
+    end
+
+    # Create the patchinfo
+    visit project_show_path(project: Project.last.name)
+    desktop? ? click_link('Create Patchinfo') : click_menu_link('Actions', 'Create Patchinfo')
+    fill_in('patchinfo_summary', with: 'ProjectWithRepo_package is much better than the old one')
+    fill_in('patchinfo_description', with: 'Fixes nothing, Fixes nothing, Fixes nothing, Fixes nothing, Fixes nothing, Fixes nothing')
+    check('patchinfo_block')
+    fill_in('patchinfo_block_reason', with: 'locked!')
+    click_button('Save')
+    expect(page).to have_css('#flash', text: 'Successfully edited patchinfo')
+
+    # Step 2: The user submits the update creating a maintenance incident
+    #####################################################################
+    visit project_show_path(project: Project.last.name)
+
+    expect(page).to have_text('patchinfo')
+
+    desktop? ? click_link('Submit as Update') : click_menu_link('Actions', 'Submit as Update')
+    expect(page).to have_title('Submit as Update')
+    fill_in('description', with: 'I want the update')
+    save_screenshot('about to submit the update.png')
+    click_button('Submit')
+
+    expect(page).to have_text('Created maintenance incident request')
+
+    logout
+
+    # Step 3: The maintenance coordinator accepts the request
+    #########################################################
+    login(maintenance_coord_user)
+
+    visit request_show_path(bs_request)
+    save_screenshot('maintenance_request.png')
+    expect(page).to have_text('This is a Maintenance Incident')
+    fill_in('reason', with: 'really? ok')
+
+    click_button('Accept request')
+    expect(page).to have_text("Request #{bs_request.number} accepted")
+
+    # Step 4: The maintenance coordinator edits the patchinfo file
+    ##############################################################
+    # FIXME: Editing patchinfos should be it's own spec...
+    visit(edit_patchinfo_path(package: 'patchinfo', project: 'MaintenanceProject:0'))
+
+    # needed for patchinfo validation
+    fill_in('patchinfo_summary', with: 'ProjectWithRepo_package is much better than the old one')
+    fill_in('patchinfo_description', with: 'Fixes nothing, Fixes nothing, Fixes nothing, Fixes nothing, Fixes nothing, Fixes nothing')
+    check('patchinfo_block')
+    fill_in('patchinfo_block_reason', with: 'locked!')
+
+    click_button('Save')
+    expect(page).to have_css('#flash', text: 'Successfully edited patchinfo')
+    expect(find(:css, '.block-reason span:first-child')).to have_text('Release is blocked')
+
+    click_link('Edit patchinfo')
+    uncheck('patchinfo_block')
+    expect(page).to have_css('input[id=patchinfo_block_reason][disabled]')
+    click_button 'Save'
+
+    logout
```

Comment:
This is another context or even another spec.

- https://www.betterspecs.org/#contexts